test/utils/objects/pod.go

```diff
@@ -37,9 +38,16 @@ func NewTestPodPause(namespace, name string) *corev1.Pod {
 			TerminationGracePeriodSeconds: &zero,
 			Containers: []corev1.Container{
 				{
-					Name:    name + "-cnt",
-					Image:   PauseImage,
-					Command: []string{PauseCommand},
+					Name:  name + "-cnt",
+					Image: images.Test(),
+					Command: []string{
```

Comment:
this change breaks the compatibility with the `pause` image, which is something I'd avoid just yet. Let's just revert this change. The function should read like
```
func NewTestPodPause(namespace, name string) *corev1.Pod {
	var zero int64
	return &corev1.Pod{
		ObjectMeta: metav1.ObjectMeta{
			Name:      name,
			Namespace: namespace,
		},
		Spec: corev1.PodSpec{
			TerminationGracePeriodSeconds: &zero,
			Containers: []corev1.Container{
				{
					Name:    name + "-cnt",
					Image:   images.GetPauseImage(),
					Command: []string{PauseCommand},
				},
			},
		},
	}
}
```